### PR TITLE
Switch to elm-live to add Apple Silicon M1 support 🤘🏿

### PR DESCRIPTION
**Pull request by vladimirlogachev**
`create-elm-app` is maintained poorly in comparison to `elm-live`.
I think it's a good switch.

But most importantly, I need these changes so that I could work on #8

### CODE REVIEW

**Review by kutyel — APPROVED** [review]
Looks good! Why remove the yarn.lock though? 🤔

**Comment by vladimirlogachev** [comment]
> Looks good! Why remove the yarn.lock though? 🤔

Haha! It's not removed. It just became smaller.

This is exactly how simpler the `elm-live` is in comparison to `create-elm-app`.

**Comment by vladimirlogachev** [comment]
I cannot merge it because I have no write access, so please do it yourself.

**Comment by kutyel** [comment]
> Haha! It's not removed. It just became smaller.
> 
> This is exactly how simpler the `elm-live` is in comparison to `create-elm-app`.

Wow, incredible in deed! 🤯

File: frontend/package.json [hunk 1]
```diff
@@ -4,14 +4,14 @@
   "description": "Elm client example for our mu-haskell GraphQL server!",
   "scripts": {
     "codegen": "elm-graphql --schema-file ../library.graphql --base LibraryApi --output src",
-    "start": "elm-app start"
+    "start": "elm-live src/Main.elm --dir=public --open --host=0.0.0.0 --port=8080 --pushstate --start-page=index.html -- --output=public/bundle.js"
```
Comment (kutyel) [inline frontend/package.json:7]: Even though the dependencies are much less heavy and the package is better maintained... this line here is terrible DX in my opinion 🤣  Why not make all those arguments the default!? 🤔

**Comment by kutyel** [comment]
Thanks again for the contribution! 🙌🏻

**Pull request merged**